src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -473,6 +474,12 @@ internal int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancella
                 return Succeeded;
             }
 
+            if (Arguments.DisplayVersion)
+            {
+                PrintVersion(consoleOutput);
+                return Succeeded;
+            }
```

Comment:
Users can combine the switches to get more than one to occur. It is a bit silly, but I don't have a problem with that.